src/styles/themes/default.js

```diff
@@ -85,6 +85,7 @@ const darkTheme = {
 darkTheme.PAGE_BACKGROUND_COLORS = {
     [SCREENS.HOME]: darkTheme.sidebar,
     [SCREENS.SETTINGS.PREFERENCES]: colors.blue500,
+    [SCREENS.SETTINGS.WALLET]: colors.green700,
```

Comment:
Add to light theme too?

Reply:
You mean `oldTheme`? I am pretty sure it doesn't have any `PAGE_BACKGROUND_COLORS` and is not even used anymore 😄